src/components/IstioConfigPreview/EditResources.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { AuthorizationPolicy, Sidecar } from 'types/IstioObjects';
+import { jsYaml } from '../../types/AceValidations';
+import AceEditor from 'react-ace';
+import { Grid, GridItem, Tab, Tabs } from '@patternfly/react-core';
+import { EditorPreview } from './EditorPreview';
+import { safeDumpOptions } from '../../types/IstioConfigDetails';
+import { style } from 'typestyle';
+import _ from 'lodash';
+
+type PolicyItem = AuthorizationPolicy | Sidecar;
+
+interface Props {
+  items: PolicyItem[];
+  orig: PolicyItem[];
+  onChange: (obj, index) => void;
+}
+
+interface State {
+  resourceTab: number;
+}
+
+export class EditResources extends React.Component<Props, State> {
+  aceEditorRef: React.RefObject<AceEditor>;
+  constructor(props: Props) {
+    super(props);
+    this.state = { resourceTab: 0 };
+    this.aceEditorRef = React.createRef();
+  }
+
+  render() {
+    return (
+      <Grid className={style({ padding: '20px' })}>
```

Comment:
I think it could be better to not use a Grid here.
I see a lot of this pattern to use a top layout with a single item, that's a little bit overkill, and we'll pay some offer when moving to another version.

In short, use a simple "div" element if you don't need anything else, but use a grid if you need a grid layout.